modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Pagination.tsx

```diff
@@ -51,14 +55,43 @@ function Pagination({
 		requiredPageSizesValidationError,
 		setRequiredPageSizesValidationError,
 	] = useState(false);
+	const [
+		invalidPageSizesLengthValidationError,
+		setInvalidPageSizesLengthValidationError,
+	] = useState(false);
 
 	const fdsViewPageSizesRef = useRef<HTMLInputElement>(null);
 	const fdsViewDefaultPageSizeRef = useRef<HTMLInputElement>(null);
 
-	const compatibilityValidation = () => {
-		const paginationArray = pageSizes.split(', ');
+	const getPageSizesArray = (): string[] => {
+		return pageSizes.trim().split(/\s*(?:,|$)\s*/);
+	};
+
+	const pageSizesFieldValidation = (pageSizesArray: string[]) => {
+		if (pageSizesArray.length > 25) {
+			setInvalidPageSizesLengthValidationError(true);
+		}
+		else {
+			setInvalidPageSizesLengthValidationError(false);
+		}
+
+		const invalidNumber = pageSizesArray.some((element) => {
+			const isNumber = /^\d+$/.test(element);
+			const pageSize: number = parseInt(element, 10);
```

Comment:
I tried with `isNaN()` and it allowed lists like `1, 2, 3wesfew, 4abc` so I wanted a dumber thing, hence the regex. This gives just positive numbers. `Number` allows much more variety (decimals, negative, scientific). Perhaps, once regex is tested we can use `Number()` instead of `parseInt`, if that is what you meant